### DISCUSSION TITLE
support Nuxt2？

### DISCUSSION THREAD

**Discussion by NMTuan** [unocss/unocss#1694] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/2149763/194686410-16a899f9-c3c0-438a-bde8-f928f0a15695.png)

when i created a new project and add unocss, it breaken. How to do it work?

### Steps to reproduce

1. use create-nuxt-app cli
```bash
yarn create nuxt-app project
```

2. add unocss with https://github.com/unocss/unocss/tree/main/packages/nuxt
```bash
yarn add -D @unocss/nuxt
```
and configure `nuxt.config.js` file.

3. yarn dev

**Comment by NMTuan** [comment]
i need to use the nuxt bridge ?

**Comment by zyyv** [comment, marked as answer]
Yes, you should use `@nuxt/bridge-edge`
```ts
import { defineNuxtConfig } from '@nuxt/bridge-edge'
```

More info please check the [example](https://github.com/unocss/unocss/blob/main/examples/nuxt2/nuxt.config.ts)
